ffi/dotnet/Devolutions.IronRdp/src/Connection.cs

```diff
@@ -23,17 +23,38 @@ public static class Connection
 
         var serverAddr = ip[0] + ":" + port;
         connector.WithServerAddr(serverAddr);
+        connector.WithDynamicChannelDisplayControl();
         if (factory != null)
         {
             var cliprdr = factory.BuildCliprdr();
             connector.AttachStaticCliprdr(cliprdr);
         }
-
+        
         await ConnectBegin(framed, connector);
         var (serverPublicKey, framedSsl) = await SecurityUpgrade(framed, connector);
         var result = await ConnectFinalize(serverName, connector, serverPublicKey, framedSsl);
         return (result, framedSsl);
     }
+    
+    public static async Task<Written> SingleSequenceStepRead<TStream>(Framed<TStream> frame, ConnectionActivationSequence sequence, WriteBuf buf)
+    where TStream: Stream
+    {
+        buf.Clear();
+
+        var pudHint = sequence.NextPduHint();
+        
+        // Don't remove, NextPduHint is generated and can return null
```

Comment:
question: What do you mean by "NextPduHint is generated"?

Reply:
The function NextPduHint() is generated, the function signature suggest that pudHint cannot be null but the actual implementation does return null. It is generated signifies that the function signature is misleading. Rider will read the signature and produce warning about pdduHint cannot be null.

Reply:
I see, thank you! The comment should be modified to clarify what is being meant.

```
// FIXME: The NextPduHint() function signature is incorrectly generated: the return value is nullable, so this check is necessary.

Reply:
updated